aries-site/src/pages/templates/lists.mdx

```diff
@@ -169,3 +170,16 @@ Two-line list with an image, primary title, and a description.
 >
   <ListImageIdentifierExample />
 </Example>
+
+### Paginated
+
+With long sets of data, pagination can help to create a more efficient means of navigating to a specific item or range of items.
+
+<Example
```

Comment:
The alignment looks off in mobile view ? maybe since more are being shown but the mobile view in table was aligned right

Reply:
Good call, on mobile it should justify center. udpating